docs/assets/css/homepage.scss

```diff
@@ -3,7 +3,7 @@
 
 // Imports -------------- //
 
-@import "us_web_design_standards";
+@import 'uswds';
```

Comment:
I'm a little confused how if can find "uswds" here. Isn't the file technically called "all.scss"? Where does this actually import from?